osgp/platform/osgp-adapter-domain-core/src/main/java/org/opensmartgridplatform/adapter/domain/core/application/services/FirmwareManagementService.java

```diff
@@ -110,6 +148,121 @@ public void handleGetFirmwareVersionResponse(final List<FirmwareVersionDto> firm
         this.webServiceResponseMessageSender.send(responseMessage);
     }
 
+    private void checkFirmwareHistory(final String organisationIdentification, final String deviceId,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "organisationIdentification". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1172)